src/condor_scripts/annex/delta.fragment

```diff
@@ -0,0 +1,21 @@
+# WELL_KNOWN_LOCATION_FOR_BINARIES=https://batlab.chtc.wisc.edu/nmi-runs/tlmiller/dist/condor-23.7.0-custom-AlmaLinux8.tar.gz
+WELL_KNOWN_LOCATION_FOR_BINARIES=https://research.cs.wisc.edu/htcondor/tarball/current/current/condor-x86_64_AlmaLinux8-stripped.tar.gz
```

Comment:
As long as we are here, can we bump the WELL_KNOWN_LOCATION_FOR_BINARIES to be "current" for all annexen, and close HTCONDOR-2165?

Reply:
No.

Delta needs the version of HTCondor that includes this PR, but that version doesn't exist yet, so it can't be downloaded. which makes it impossible to actually test Delta support with the released code.  Downloading the 'current' version is a work-around for this problem.  Once that version does exist, this URL should be updated to the specific version that I know works and not change without a good reason: we don't test `htcondor annex` except by hand on the various systems it supports, so to suggest that any random future version of HTCondor is going to work is just asking for a support nightmare.

Reply:
Works for me.